src/Pages/LandingPage/LandingComponents/FeaturedContainer/BrandSection/BrandSection.jsx

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+
+export const BrandSection = () => {
+  const [brandData, setBrandData] = useState([]);
+
+  useEffect(() => {
+    (async function () {
+      const { data } = await axios.get("/api/brands");
+      setBrandData(data.brands);
+    })();
+  }, []);
+
+  return (
+    <div className="categories-section">
+      {brandData.map((brands) => {
```

Comment:
Here, we can improve the code a little bit.
As there is only two property in the object, we can use destructuring here. Like instead of `brands`, we can do `{id, image}`

Reply:
Thanks for pointing it out I will make the change.